x/denommetadata/types/tonemmetadata.go

```diff
@@ -0,0 +1,64 @@
+package types
+
+import (
+	"fmt"
+
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+)
+
+func NewTokenMetadata(description string, denomstring string, exponent uint32, denomalias string, base string, display string, name string, symbol string, uri string, urihash string) *TokenMetadata {
```

Comment:
redundant file if u use the bank denommetadata type

Reply:
fixed